src/calculateRentalCost.js

```diff
@@ -29,6 +29,18 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const rentCost = 40;
+  const totalRentCost = rentCost * days;
+
+  if (days >= 3 && days < 7) {
+    return totalRentCost - 20;
+  }
+
+  if (days >= 7) {
+    return totalRentCost - 50;
+  }
+
+  return totalRentCost;
```

Comment:
```suggestion
  if (days >= 7) {
    return totalRentCost - 50;
  }
  
  if (days >= 3) {
    return totalRentCost - 20;
  }

  return totalRentCost;
```
Если поменять их местами - можно упростить условие в if

Reply:
3, 7, 20 и 50 тоже вынеси в переменные